CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- Added official support for PHP 8.1 [#74] from [@grogy]
```

Comment:
```suggestion
- GH Actions: testing against PHP 8.2  [#74] from [@grogy]
```
... and moving this to "Internal"

"Official support" IMO is:
* Actually working version (which we currently don't have for the Phar - see #69 and #70).
* Indicated via the Composer `php` version constraints.

This PR just changes the CI.

Reply:
Thank you, I will change the changelog.

I mean constrainst are correct because is set only minimal PHP version and maximal is undefined